Backend/prisma/migrations/20240720022753_init_services_model/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
+CREATE TABLE "user" (
+    "id" TEXT NOT NULL,
+    "email" TEXT NOT NULL,
+    "phoneNumber" TEXT NOT NULL,
+    "hashedPassword" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "userType" TEXT NOT NULL,
+    "profilePhoto" BYTEA,
+    "userAddress" TEXT NOT NULL,
+
+    CONSTRAINT "user_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "serviceProvider" (
+    "id" TEXT NOT NULL,
+    "email" TEXT NOT NULL,
+    "businessName" TEXT NOT NULL,
+    "businessAddress" TEXT NOT NULL,
+    "phoneNumber" TEXT NOT NULL,
+    "hashedPassword" TEXT NOT NULL,
+    "userType" TEXT NOT NULL,
+    "services" TEXT NOT NULL,
+    "profilePhoto" BYTEA,
+    "bio" TEXT NOT NULL,
+    "priceRange" TEXT NOT NULL,
```

Comment:
should price be of type text? just wondering - not blocking, but if we make any tables in the future let's think about the type!